app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ProfileScreenTest.kt

```diff
@@ -0,0 +1,172 @@
+package com.github.se.cyrcle.ui.profile
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.navigation.compose.rememberNavController
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.se.cyrcle.ui.navigation.NavigationActions
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ProfileScreenTest {
+  @get:Rule val composeTestRule = createComposeRule()
+
+  private fun setupProfileScreen() {
```

Comment:
Here this function could be annotated with `@Before` and removed from the body of other `@Test` functions.
The `@Before` makes it that it is called automatically before every test, so that you don't have to call it in your functions.